apps/freelance/src/redux/services/talents.ts

```diff
@@ -0,0 +1,16 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const talentApi = createApi({
+  reducerPath: 'talentApi',
+  baseQuery: fetchBaseQuery({
+    baseUrl:
+      'https://randomuser.me/api/?results=10&inc=name,gender,email,nat,picture&noinfo',
```

Comment:
I already left comments about the links in the code, its unacceptable!

Reply:
fixed